doc/uftrace-live.md

```diff
@@ -50,6 +46,39 @@ OPTIONS
     explicitly have 'trace' trigger, those are always traced regardless of
     execution time.
 
+-S *SCRIPT_PATH*, \--script=*SCRIPT_PATH*
+:   Add a script to do additional work at the entry and exit of function.
+    The type of script is detected by the postfix such as '.py' for python.
+
+\--no-libcall
+:   Do not record library function invocations.  Library calls are normally
+    traced by hooking the dynamic linker's resolve function in the PLT.  One can
+    disable it with this option.
+
+\--no-event
+:   Disable event recording which is used by default.  Note that explicit event
+    tracing by \--event option is not affected by this.
+
+\--match=*TYPE*
+:   Use pattern match using TYPE.  Possible types are `regex` and `glob`.
+    Default is `regex`.
+
+\--disable
+:   Start uftrace with tracing disabled.  This is only meaningful when used with
+    a `trace_on` trigger.
+
+
+LIVE OPTIONS
+============
+\--list-event
+:   Show available events in the process.
+
+\--report
+:   Show live-report before replay.
```

Comment:
I'm not so sure if they have to be in another category, but just wanted to talk about it what to do.

Reply:
I'm fine with the current state.